### PR TITLE
[docker] add service start

### PR DESCRIPTION
**Pull request by gjc13**

### CODE REVIEW

File: script/docker_entrypoint.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@
 #  3. Neither the name of the copyright holder nor the
 #     names of its contributors may be used to endorse or promote products
 #     derived from this software without specific prior written permission.
-#
+#
```
Comment (bukepo) [inline script/docker_entrypoint.sh]: nit: extra space.

File: third_party/wpantund/wpantund.init [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            log_progress_msg "wpantund running"
+            log_end_msg 0
+        else
+            log_progress_msg "wpantund not running"
+            log_end_msg 1
+            exit 1
+        fi
+        ;;
+    *)
+        echo "unkown comand" >&2
```
Comment (bukepo) [inline third_party/wpantund/wpantund.init]: typo

File: script/server [hunk 3]
```diff
@@ -38,16 +38,23 @@ main()
...
     sudo sysctl --system
+    if have systemctl; then
+        systemctl start dbus
+    else
+        service dbus start
+    fi
     nat64_start || die 'Failed to start NAT64!'
     dns64_start || die 'Failed to start DNS64!'
     if have systemctl; then
+        systemctl is-active dbus || sudo systemctl start dbus || die 'Failed to start wpantund!'
```
Comment (bukepo) [inline script/server]: `wpantund` -> `dbus`

File: script/server [hunk 4]
```diff
@@ -38,16 +38,23 @@ main()
...
         systemctl is-active wpantund || sudo systemctl start wpantund || die 'Failed to start wpantund!'
         systemctl is-active otbr-web || sudo systemctl start otbr-web || die 'Failed to start otbr-web!'
         systemctl is-active otbr-agent || sudo systemctl start otbr-agent || die 'Failed to start otbr-agent!'
     elif which service; then
-        sudo service wpantund start  || die 'Failed to start wpantund!'
-        sudo service otbr-web start || die 'Failed to start otbr-web!'
-        sudo service otbr-agent start || die 'Failed to start otbr-agent!'
+        sudo service dbus status || sudo service dbus start  || die 'Failed to start wpantund!'
```
Comment (bukepo) [inline script/server]: wpantund -> dbus

File: script/server [hunk 5]
```diff
@@ -38,16 +38,23 @@ main()
 {
     . $BEFORE_HOOK
     sudo sysctl --system
+    if have systemctl; then
```
Comment (bukepo) [inline script/server]: duplicate with commands blow?

File: src/agent/Makefile.am [hunk 6]
```diff
@@ -132,37 +132,69 @@ EXTRA_DIST                = \
...
     otbr-agent.service                \
     $(NULL)
+endif
+
+initddir=$(sysconfdir)/init.d
 
 defaultdir=$(sysconfdir)/default
 
-install-data-local: $(DESTDIR)$(defaultdir)/otbr-agent
+uninstall-local: uninstall-default uninstall-initd
```
Comment (bukepo) [inline src/agent/Makefile.am]: -> `uninstall-local: uninstall-default`
and
move `uninstall-local: uninstall-initd` inside of `if OTBR_HAS_SYSTEMCTL`

**Comment by gjc13** [comment]
@bukepo @jwhui Any more comments?

File: third_party/wpantund/Makefile.am [hunk 7]
```diff
@@ -21,7 +21,7 @@
 #  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
 #  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
 #  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
-#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
```
Comment (bukepo) [inline third_party/wpantund/Makefile.am]: revert this change?
